### PR TITLE
Service: Anemometer

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request adds a new service definition for JACDAC.

### CODE REVIEW

File: services/0x1b591bbf.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Anemometer
```
Comment (mmoskal) [inline services/0x1b591bbf.md:1]: the file should be called anemometer.md

**Comment by pelikhan** [comment]
Yeah was a big in my PR generator

________________________________

File: services/0x1b591bbf.md (hunk 1 above)
Comment (pelikhan) [reply services/0x1b591bbf.md:1]: my pr generator messed up

File: services/0x1b591bbf.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Registers
+
+    ro wind_speed: u16.16 m/s @ reading
+
+The velocity of the wind.
+
+    const max_wind_speed?: u16.16 m/s @ max_reading
+
+Maximum speed that can be measured by the sensor.
```
Comment (pelikhan) [inline services/0x1b591bbf.md:16]: needs error too

File: services/0x1b591bbf.md (hunk 1 above)
Comment (pelikhan) [reply services/0x1b591bbf.md:1]: wind speed is easier for everyone

**Pull request merged**